python/llm/test/run-llm-check-function.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# ipex may return exit code 127, which cause unexpected error
+# ref: https://github.com/intel/intel-extension-for-pytorch/issues/634
+pytest_check_error() {
+  result=$(eval "$@" || echo "FINISH PYTEST")
+  echo $result > pytest_check_error.log
+  cat pytest_check_error.log
+  failed_lines=$(cat pytest_check_error.log | { grep failed || true; })
+  if [[ $failed_lines != "" ]]; then
+    exit 1
+  fi
+  rm pytest_check_error.log
+}
+
+# ipex may return exit code 127, which cause unexpected error
+# ref: https://github.com/intel/intel-extension-for-pytorch/issues/634
+ipex_workaround_wrapper() {
+    eval "$@" || ( [[ $? == 127 && $RUNNER_OS == "Windows" ]] && echo "EXIT CODE 127 DETECTED ON WINDOWS, IGNORE." || exit 1)
+}
```

Comment:
Maybe add some comment here that `pytest_check_error` is for tests using `pytest ...`, and the other is for tests using `python ...`, so that it will be more clear for developers to add further tests.